lib/Doctrine/Validator.php

```diff
@@ -108,6 +108,10 @@ public static function validateLength($value, $type, $maximumLength)
             $decimalPoint = $localeInfo['mon_decimal_point'] ? $localeInfo['mon_decimal_point'] : $localeInfo['decimal_point'];
             $e            = explode($decimalPoint, (string) $value);
 
+            if (!$e) {
+                $e = array();
+            }
+
             $length = strlen($e[0]);
```

Comment:
I'm surprised the analyzers didn't complain here, since $e can be set to an empty array now, wouldn't `$e[0]` be undefined?

Reply:
Yeah, good point, PHP does throw an undefined offset Notice in this case. I can update to check the length of `$e`.